src/Presentation/Http/Requests/README.md

```
@@ -0,0 +1,23 @@
+class RegisterRequest extends FormRequest
+{
+    public function rules(): array
+    {
+        return [
+            'username' => ['required', 'string', 'min:3', 'max:20', 'unique:users'],
+            'email' => ['required', 'email', 'unique:users'],
+            'password' => ['required', 'string', 'min:8', 'confirmed'],
+        ];
+    }
+}
+```
```

markdownlint-cli2 (0.17.2)

Files should end with a single newline character
null

(MD047, single-trailing-newline)